migrations/20231007120000_create_tiny_url.up.sql

```diff
@@ -0,0 +1,14 @@
+BEGIN;
+
+CREATE TABLE tiny_url (
+  id bigserial PRIMARY KEY,
+  org_url text NOT NULL,
+  short_url text UNIQUE NOT NULL,
+  comment text,
+  user_id int NOT NULL,
+  expired_at timestamp NOT NULL,
+  created_at timestamp DEFAULT (NOW() AT TIME ZONE 'UTC'),
+  created_by text NOT NULL
```

Comment:
- will `user_id` not be enough to identify the creator of this link?

Reply:
That is asked by @iamitprakash like if he want the name who created the URL then by this URL table we can get it

Reply:
- Will this not create redundancy? Also if `user_id` is the primary key in the users table a lookup for a username will not be expensive. What do you think @iamitprakash ?

Reply:
I don't think so. we have separate plan for user migration as well